src/Symfony/Component/Console/Application.php

```diff
@@ -1095,7 +1095,19 @@ private function splitStringByWidth($string, $width)
         $utf8String = mb_convert_encoding($string, 'utf8', $encoding);
         $lines = array();
         $line = '';
-        foreach (preg_split('//u', $utf8String) as $char) {
+
+        $iterator = static function ($string) {
+            while (strlen($string)) {
```

Comment:
Shouldn't be `mb_strlen` instead?

Reply:
`'' !== $string` would be even better

Reply:
I just want it to be non-empty, so `mb_strlen` and `strlen` will both return `0` for empty string, and a `true`-value when non-empty. But I can add a `> 0` to have a `bool` inside the `if` instead of an `int`

Reply:
Fixed